design/FY2022/enhacement_to_resilience_metric.md

```diff
@@ -0,0 +1,191 @@
+|Space<sub>1</sub>||||
+|...||||
+|Space<sub>N</sub>||||
+|Min||||
+|Max||||
+|Average||||
+|Sum||||
+<p style="text-align: center;"> Table 4. Sample report table of discomfort-weighted exceedance hours.</p>
+
+When *ThermalResilienceSummary* is declared in *Output:Table:SummaryReports*, the above three tables will be generated and presented in the tabular reports. In particular, the *Hours of Safety* table will only be presented if an *Output:Table:SummaryReports:ReportPeriod* object is defined for ThermalResilienceSummary report. 
+
+## Testing/Validation/Data Source(s): ##
+
+Two example files (the DOE reference small office and the one zone uncontrolled model) will be modified to demonstrate the use of the new feature. Simulation results will be manually checked/benchmarked using an excel spreadsheet with input and output from EnergyPlus runs.
+
+## IDD Object changes: ##
+
+Two extra fields will be added to the current People object to define the safe temperature thresholds for a certain kind of population.
+
+    People,
+      N7, \field Cold Stress Temperature Threshold
+          \type real
+          \units C
+          \note this is the indoor safe temperature threshold for cold stress
+    \default 15.56
+      N8, \field Heat Stress Temperature Threshold
+          \type real
+          \units C
+          \note this is the indoor safe temperature threshold for heat stress
+          \default 30
+
+We will add a new object *Output:Table:SummaryReports:ReportPeriod* to add supplementary information to define the reporting period of certain summary tables.
+
+    Output:Table:SummaryReports:ReportPeriod,
+        \memo This object allows the user to add a reporting period to a certain tabular report when the reporting period is not the whole simulation period. When defined, the declared summary report is based on aggregations from the start date to the end date defined. 
+      A1, \field Report Name,
+          \key ThermalResilienceSummary
+          \key CO2ResilienceSummary
+          \key VisualResilienceSummary
+        \key …
+      N1 , \field Begin Month
+          \required-field
+          \minimum 1
+          \maximum 12
+          \type integer
+      N2 , \field Begin Day of Month
+          \required-field
+          \minimum 1
+          \maximum 31
+          \type integer
+      N3 , \field Begin Hour of Day
+          \required-field
+          \minimum 1
+          \maximum 24
+          \type integer
+      N4 , \field End Month
+          \required-field
+          \minimum 1
+          \maximum 12
+          \type integer
+      N5 , \field End Day of Month
+          \required-field
+          \minimum 1
+          \maximum 31
+          \type integer
+      N6 , \field End Hour of Day
+          \required-field
+          \minimum 1
+          \maximum 24
+          \type integer
+
+## Proposed additions to Meters: ##
+
+N/A
+
+## Proposed Report Variables: ##
```

Comment:
Would any of these new measure be useful as a time series output variable?

Reply:
Maybe some variables like "unmet degree hours" and "discomfort weighted exceedance hours" could potentially be useful as a time series output. Hours of safety may be less meaningful as a standalone time series output, as it measures _the time_ a zone first crosses the dangerous temperature threshold. However, the implementation timeline is a bit tight. We're trying to start simple. Maybe the addition of other time series output variables could be made in the next feature update?

Reply:
@yujiex Of course, it's fine to wait to add output variables later. And since you are on a tight schedule, the zone daylighting question can wait also.

Reply:
Thanks, @mjwitte. We'll leave this time series variable reporting and the daylighting part to future development.

Reply:
Hi @mjwitte, we were considering adding some variables to the advanced report variable set for debugging purposes.